### ISSUE TITLE
Uncaught AttributeError in sendchat.py line 30

### ISSUE THREAD

**Issue by KashifManzoor64** [paul-gauthier/aider#1892]
Aider version: 0.54.7

An uncaught exception occurred:

```
Traceback (most recent call last):
  File "runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "__main__.py", line 7, in <module>
    sys.exit(main())
  File "main.py", line 640, in main
  File "base_coder.py", line 727, in run
    while True:
  File "base_coder.py", line 770, in run_one
  File "base_coder.py", line 1157, in send_message
    if messages[-1]["role"] == "assistant":
  File "base_coder.py", line 1835, in auto_commit
    self.reflected_message = str(err)
  File "repo.py", line 101, in commit
    return
  File "repo.py", line 181, in get_commit_message
    content += diffs
  File "sendchat.py", line 38, in wrapper
    retry_exceptions(),
  File "sendchat.py", line 30, in retry_exceptions
    litellm.llms.anthropic.chat.AnthropicError,
AttributeError: module 'litellm.llms.anthropic' has no attribute 'AnthropicError'

```

**Comment by KashifManzoor64** [comment]
Faced above issue while creating the files

**Comment by fry69** [comment]
Thank you for filing this issue.

The fix is in the latest version (`0.58.1`), please update aider with
```
aider --upgrade
```
Please do not forget to exit and restart aider after it updated itself.

Duplicate #1278
